src/style.css

```diff
@@ -0,0 +1,22 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.red {
+  background-color: red;
+  height: 100px;
+  width: 100%;
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) - do not repeat code

Reply:
create one general class name for repeated styles